Changes.md

```diff
@@ -30,6 +30,7 @@ Fixes
 
 - Cycles : Fixed issue where scaling unnormalized quad and disk lights would not affect their brightness.
 - ShaderView : Fixed crash caused by a SceneCreator returning `None`. [^1]
+- Instancer : Support 64 bit ints for ids ( matching what is loaded from USD ).
```

Comment:
This needs moving to the section for the next release, and rewording to use the past tense.